stages/org.osbuild.uki

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python3
+"""
+Create a Unified Kernel Image
+
+A Unified Kernel Image (UKI) is an UEFI executable that contains
+an UEFI stub loader, together with the kernel, the initrd, the
```

Comment:
`a UEFI`, not `an UEFI`, and I am not accepting any wikipedia-links that argue otherwise!

Reply:
ok ok :)